packages/svelte/src/compiler/phases/3-transform/client/visitors/template.js

```diff
@@ -2070,12 +2071,11 @@ export const template_visitors = {
 			}
 		};
 
+		namespace = node.metadata.svg ? namespace_svg : null;
```

Comment:
This should happen when the variable is declared (and maybe it can be turned into a `const` then if it isn't reassigned later).

Also, it's probably better to use `b.id('$.namespace_svg')` here which means the constant is reused across places instead of having the string written out - should minify better overall.